src/lib/server/editors/data/DataSuggestionEditor.ts

```diff
@@ -0,0 +1,22 @@
+import { PromptEditor } from '$lib/server/editors/editor';
+import type { Tables } from '$lib/supabase';
+import { EditorType } from '$lib/types';
+
+export class DataSuggestionEditor extends PromptEditor {
+	constructor(suggestion: string) {
+		super(
+			'DataSuggestion',
+			'Generate Test Cases',
+			'Generate new test cases for the following category: ' + suggestion,
```

Comment:
nit: use string interpolation instead? don't know how we want to do this if it's at the end anyway. Maybe fine also.

Reply:
I think at end appending is fine imo